nuxeo/nuxeo.py

```diff
@@ -269,26 +313,28 @@ def execute(
         if action and action.progress is None:
             action.progress = 0
 
-        if file_out is not None:
+        if file_out:
             locker = self.unlock_path(file_out)
             try:
-                with open(file_out, 'wb') as f:
-                    while 'downloading':
+                with open(file_out, str('wb')) as f:
+                    chunk_size = self.download_buffer_size
+                    for chunk in resp.iter_content(chunk_size=chunk_size):
```

Comment:
Use `resp.iter_content(chunk_size=chunk_size, b'')` and remove later check:

    if chunk == '':
        break